bin/build

```diff
@@ -17,7 +17,14 @@ while [ -z ${halyard_armory_version} ]; do
             break
         fi
     done
+
     PAGE_NUM=$((${PAGE_NUM} + 1))
+
+
+    # let's not go too crazy
+    if [ $PAGE_NUM -gt 10 ]; then
+      break
```

Comment:
shouldn't we just set halyard_armory_version to 0.0.0 here? then the while loop condition will fail and break out of the loop there? can also be explicit and set the version here AND break

Reply:
oh whoops, I forgot to include the previous chunk in my commit/PR.